### DISCUSSION TITLE
[Design/Race] Moths

### DISCUSSION THREAD

**Discussion by S34NW** [ParadiseSS13/Paradise#16992] [Documents]
# Mothpeople come to paradise

### Sources

Primary spriting and code inspiration can be ported from TG, however we will not be copying them line for line.
Original TG PR (many follow up fixes and changes not included in this) https://github.com/tgstation/tgstation/pull/34498

### Desired outcome

A new karma locked playable race, with a unique balance set to differentiate them from other species.

## Design

Mothpeople should not feel like reskinned humans, we want unique races that do things a little differently. As such, some custom traits and a bit of a different design to "laser do more damage" was attempted.

30 Karma point cost.

### Boons

- Darksight (5m)
- Wings allow easy navigation in pressurised zero gravity scenarios
- Broken bones do not cause slowdown if wings are intact
- Cocoon state
  - 5 Second wind-up
  - Sleep for 15 seconds in cocoon
    - If you emerge naturally after 15s, you restore burnt wings and antennae
    - If you emerge early due to destroyed cocoon, you take 50 brute and burn damage, and are weakened
    - Emerging from a cocoon uses 200 nutrition, enough to take you down two or so nutrition levels

### Maluses

- Increased toxin damage
- Wings burn off when on fire
- Eyes additionally vulnerable to flashes
- Staying too long in strong light causes confusion (go away lamp meme)
  - Strong light is standing too close to multiple light tubes in day lighting mode, for example
- Flyswatters do 10 damage, not 1

## Feasability

Due to using common body shapes, no additional sprites for clothes will be needed.

Sprites for wings, antennae, and body markings are required. These can be ported from TG and users can tweak them after implementation.

Hardsuits and clothing do not block wings or antennae.

Custom caterpillar monkey form would be easy to implement.

Slight CPU use due to checking light levels on life().

Appearence:
![image](https://user-images.githubusercontent.com/12197162/139328152-c99d2d71-9a42-4d5e-97cb-edd3f9db6b96.png)

### Moth poster for motivation to read this

![image](https://user-images.githubusercontent.com/12197162/138678437-6ac696d0-0cea-4a52-9b41-3298aa4f6c26.png)

**Comment by roarbark** [comment]
looks good! i would clarify what you mean with "strong light" more though, would that be standing next to a light tube? this would be cool cuz you could have the head of the department accomodate to the mothperson by putting some lights on night lighting.

i think the interesting part is going to be lore, tg apparently has them a nomad race, working and sending resources to a centralized fleet that is looking for a new homeworld since the old one was "lost", as far as i know. their language seems to be leaned on scandinavian (tg calls it completely unintelligible), so we should have naming schemes around that, too, because on tg they only have old and stale 'moff' joke kinda names

**Reply by S34NW** [reply]
I clarified lights, thanks for that. In terms of lore and such, I am not the best with that so cannot really offer suggestions. Language syllables and naming standards is another that that could be decided by people

**Comment by Latentish** [comment]
GIVE ME THE MOTH.

**Comment by Ionward** [comment]
Will there be a way for moths to regenerate their wings, like if they used their cocoon or something? While uncommon, its pretty easy to catch on fire if anyone else does near you, and losing one of their main boons with no way to work to get it back sounds pretty rough (but not a dealbreaker)

**Reply by roarbark** [reply]
tg had a wing reconstruction surgery iirc

**Comment by Miraviel** [comment]
Will there be a way to replace their burnt off wings? Via surgery... cybernetics or the like?

**Comment by Fox-McCloud** [comment]
I'm not sure I can or will support this.

**Reply by Latentish** [reply]
Any particular reason why?

**Comment by hal9000PR** [comment]
> Wings burn off when on fire


immolator power creep
